licenses/H5Zbzip2_LICENSE.txt

```diff
@@ -0,0 +1,29 @@
+Copyright Notice and Statement for PyTables Software Library and Utilities:
```

Comment:
Delete these and just link `src/filters/H5Zlz4/THIRDPARTY.txt` in the readme. Anyone copying the  `HDF5.jl` would have them included so this would be redundant.

Reply:
I deleted these in 2ab4410 and moved the readme to THIRDPARTY.md at the root level.